### PR TITLE
feat: resize pvc

### PR DESCRIPTION
**Pull request by jiuker**
fix https://github.com/minio/operator/issues/1780

### CODE REVIEW

**Review by shtripat — APPROVED** [review]
LGTM. Remove the debug prints.

**Review by shtripat — DISMISSED** [review]
lgtm

**Review by pjuarezd — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

File: pkg/controller/pvc.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package controller
```
Comment (harshavardhana) [inline pkg/controller/pvc.go:1]:
```suggestion
package controller
```

missing copyright header for this source file.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: pkg/controller/pvc.go (hunk 1 above)
Comment (allanrogerr) [reply pkg/controller/pvc.go:15]: @jiuker FYI

**Review by harshavardhana — APPROVED** [review]

**Review by pjuarezd — CHANGES_REQUESTED** [review]
I am not sure this is good, what is preventing  Operator from increase infinite times the Storage?

CRD's and resources are declarative ways to ask `what` is expected, the `additionalStorage` field seems a procedural one.
ie, declarative is: `Give me a 10GB PV`, this other seems procedural `Give me 10GB PV, then add other 5GB`.

I think this should be revaluated @jiuker, can the PV's be resized?, ie: I know DirectPV can [Expand Volume](https://github.com/minio/directpv/blob/master/docs/volume-management.md#expand-volume), I think that is what we should be suggesting to users instead of workaround it in Operator.

**Comment by jiuker** [comment]
> I am not sure this is good, what is preventing Operator from increase infinite times the Storage?
> 
> CRD's and resources are declarative ways to ask `what` is expected, the `additionalStorage` field seems a procedural one. ie, declarative is: `Give me a 10GB PV`, this other seems procedural `Give me 10GB PV, then add other 5GB`.
> 
> I think this should be revaluated @jiuker, can the PV's be resized?, ie: I know DirectPV can [Expand Volume](https://github.com/minio/directpv/blob/master/docs/volume-management.md#expand-volume), I think that is what we should be suggesting to users instead of workaround it in Operator.

The intention here is simply to execute a command similar to expanding a PVC without causing the STS to restart. Many STS scaling operations typically result in pod restarts. Alternatively, we could use a MinioJob to accomplish this task. If this is possible, let's close this PR. @pjuarezd

**Pull request closed without merging**